sql_metadb/derived_tables/loans_renewal_count.sql

```diff
@@ -0,0 +1,35 @@
+DROP TABLE IF EXISTS loans_renewal_count;
+
+-- Create a derived table that contains all items from inventory_items
+-- and adds circulation info (total loans and renewals) from
+-- circulation_loans; add in a date to show when the report was last
+-- run and fill circulation nulls with zero.
+CREATE TABLE loans_renewal_count AS
+WITH loan_count AS (
+    SELECT
+        item_id::uuid,
+        count(DISTINCT id::uuid) AS num_loans,
+        sum(renewal_count::bigint) AS num_renewals
```

Comment:
The function _sum(bigint)_ returns _numeric_; so I am going to add a cast to _bigint_.  Reference: https://www.postgresql.org/docs/current/functions-aggregate.html